napari_clusters_plotter/_measure.py

```diff
@@ -10,6 +10,7 @@
 from napari_tools_menu import  register_dock_widget
 
 @register_dock_widget(menu="Measurement > Measure intensity, shape and neighbor counts (ncp)")
+
```

Comment:
Is there a particular reason for adding this empty line here?

Reply:
not really :D

Reply:
I will remove the line later anyway because my perfectionist self agrees to let PyCharm reformat files so it removes/adds lines where needed 😂